datatableview/columns.py

```diff
@@ -352,7 +359,8 @@ def search(self, model, term):
                             k = '%s__exact' % (sub_source,)
                             column_queries.append(Q(**{k: str(db_value)}))
 
-                lookup_types = handler.get_lookup_types()
+                if not lookup_types:
```

Comment:
The else part that I commented out seemed wrong to me (as it says if term being searched is a single word ignore it for the in lookup..). I did not mean to include it in this commit, but realized later on. Should be kept if OK. Search functionality worked fine either way in my experience.